guides/training_keras_models_on_cloud.py

```diff
@@ -1,421 +1,244 @@
+        </a>
+  </td>
+</table>
 
-"""
-## Setup
+TensorFlow Cloud is a library that makes it easier to do training and hyperparameter tuning of Keras models on Google Cloud.
 
-We'll get started by installing TensorFlow Cloud, and importing the packages we
-will need in this guide.
-"""
+Using TensorFlow Cloud's `run` API, you can send your model code directly to your Google Cloud account, and use Google Cloud compute resources without needing to login and interact with the Cloud UI (once you have set up your project in the console).
+
+This means that you can use your Google Cloud compute resources from inside directly a Python notebook: a notebook just like this one! You can also send models to Google Cloud from a plain `.py` Python script.
 
-"""shell
-pip install -q tensorflow_cloud
+## Simple example
+
+This is a simple introductory example to demonstrate how to train a model remotely using [TensorFlow Cloud](https://tensorflow.org/cloud) and Google Cloud.
+
+You can just read through it to get an idea of how this works, or you can run the notebook in Google Colab. Running the notebook requires connecting to a Google Cloud account and entering your credentials and project ID. See [Setting Up and Connecting To Your Google Cloud Account](https://github.com/tensorflow/cloud/blob/master/g3doc/tutorials/google_cloud_project_setup_instructions.ipynb) if you don't have an account yet or are not sure how to set up a project in the console.
+
+## Import required modules
 """
 
 import tensorflow as tf
-import tensorflow_cloud as tfc
+tf.version.VERSION
 
-from tensorflow import keras
-from tensorflow.keras import layers
+! pip install -q tensorflow-cloud
+import tensorflow_cloud as tfc
+print(tfc.__version__)
 
-"""
-## API overview: a first end-to-end example
-
-Let's begin with a Keras model training script, such as the following CNN:
-
-```python
-(x_train, y_train), (x_test, y_test) = keras.datasets.mnist.load_data()
-
-model = keras.Sequential(
-    [
-        keras.Input(shape=(28, 28)),
-        # Use a Rescaling layer to make sure input values are in the [0, 1] range.
-        layers.experimental.preprocessing.Rescaling(1.0 / 255),
-        # The original images have shape (28, 28), so we reshape them to (28, 28, 1)
-        layers.Reshape(target_shape=(28, 28, 1)),
-        # Follow-up with a classic small convnet
-        layers.Conv2D(32, 3, activation="relu"),
-        layers.MaxPooling2D(2),
-        layers.Conv2D(32, 3, activation="relu"),
-        layers.MaxPooling2D(2),
-        layers.Conv2D(32, 3, activation="relu"),
-        layers.Flatten(),
-        layers.Dense(128, activation="relu"),
-        layers.Dense(10),
-    ]
-)
+import sys
 
-model.compile(
-    optimizer=keras.optimizers.Adam(),
-    loss=keras.losses.SparseCategoricalCrossentropy(from_logits=True),
-    metrics=keras.metrics.SparseCategoricalAccuracy(),
-)
+"""## Project Configurations
```

Comment:
To ensure correct parsing of the file, section titles should be structured as:

```
"""
## Title
```

(add a line break before each section title)

Reply:
Done.